### DISCUSSION TITLE
Is there a way to have an interactive Google Map in a Flet web app?

### DISCUSSION THREAD

**Discussion by eprparadocs** [flet-dev/flet#1284] [Q&A]
### Question

The title says it all: I need to have an interactive Google Map, which I can do in Flutter, in FLET. Is it at all possible and if so, how?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
There is a control in works: https://github.com/flet-dev/flet/pull/1122

**Reply by eprparadocs** [reply]
thanks for the information. Do you have any idea as to when it will be available?

**Comment by FeodorFitsner** [comment]
Let's give some time the author of that PR to amend it before merge.

**Reply by HeeebsInc** [reply]
Is there any update on when this will become available? Currently, fort checks all of boxes except for interactive mapping
